Octokit.Tests.Integration/Clients/OrganizationClientTests.cs

```diff
@@ -0,0 +1,124 @@
+            public async Task ReturnsCorrectCountOfOrganizationsWithStart()
+            {
+                string orgLogin1 = Helper.MakeNameWithTimestamp("MyOrganization1");
+                string orgName1 = string.Concat(orgLogin1, " Display Name 1");
+                string orgLogin2 = Helper.MakeNameWithTimestamp("MyOrganization2");
+                string orgName2 = string.Concat(orgLogin2, " Display Name 2");
+
+                var newOrganization1 = new NewOrganization(orgLogin1, EnterpriseHelper.UserName, orgName1);
+                var newOrganization2 = new NewOrganization(orgLogin2, EnterpriseHelper.UserName, orgName2);
+                await _github.Enterprise.Organization.Create(newOrganization1);
+                await _github.Enterprise.Organization.Create(newOrganization2);
+
+                var options = new ApiOptions
+                {
+                    PageSize = 1,
+                    PageCount = 1,
+                    StartPage = 2,
+                };
+
+                var organizations = await _organizationsClient.GetAllForCurrent(options);
+
+                Assert.Equal(1, organizations.Count);
+            }
+
+            [GitHubEnterpriseTest]
+            public async Task ReturnsDistinctResultsBasedOnStartPage()
+            {
+                string orgLogin1 = Helper.MakeNameWithTimestamp("MyOrganization1");
+                string orgName1 = string.Concat(orgLogin1, " Display Name 1");
+                string orgLogin2 = Helper.MakeNameWithTimestamp("MyOrganization2");
+                string orgName2 = string.Concat(orgLogin2, " Display Name 2");
+                string orgLogin3 = Helper.MakeNameWithTimestamp("MyOrganization3");
+                string orgName3 = string.Concat(orgLogin3, " Display Name 3");
+
+                var newOrganization1 = new NewOrganization(orgLogin1, EnterpriseHelper.UserName, orgName1);
+                var newOrganization2 = new NewOrganization(orgLogin2, EnterpriseHelper.UserName, orgName2);
+                var newOrganization3 = new NewOrganization(orgLogin3, EnterpriseHelper.UserName, orgName3);
+                await _github.Enterprise.Organization.Create(newOrganization1);
+                await _github.Enterprise.Organization.Create(newOrganization2);
+                await _github.Enterprise.Organization.Create(newOrganization3);
+
+                var startOptions = new ApiOptions
+                {
+                    PageSize = 1,
+                    PageCount = 1
+                };
+
+                var firstPage = await _organizationsClient.GetAllForCurrent(startOptions);
+
+                var skipStartOptions = new ApiOptions
+                {
+                    PageSize = 1,
+                    PageCount = 1,
+                    StartPage = 2
+                };
+
+                var secondPage = await _organizationsClient.GetAllForCurrent(skipStartOptions);
+
+                Assert.NotEqual(firstPage[0].Name, secondPage[0].Name);
```

Comment:
`.Name` is `null` on - you should use `.Login` field and then everything is green!  :tada: